.github/workflows/pull-request-checks.yaml

```diff
@@ -242,6 +242,76 @@ jobs:
       - name: Run tests
         run: cd build; ctest . -V -L CORE -j2
 
+  # This job takes approximately N minutes
```

Comment:
N=34?

Reply:
My apologies for failing to clean this up, I've now taken care of this in https://github.com/diffblue/cbmc/pull/7504.